File: SDKs/twilio.com/twilio_trusthub_v1/1.36.0/go/pkg/models/operations/deletecustomerprofileentityassignment.go
package operations

import (
	"openapi/pkg/models/shared"
)

var DeleteCustomerProfileEntityAssignmentServers = []string{
	"https://trusthub.twilio.com",
}

type DeleteCustomerProfileEntityAssignmentPathParams struct {
	CustomerProfileSid string `pathParam:"style=simple,explode=false,name=CustomerProfileSid"`
	Sid                string `pathParam:"style=simple,explode=false,name=Sid"`
}

type DeleteCustomerProfileEntityAssignmentSecurity struct {
	AccountSidAuthToken shared.SchemeAccountSidAuthToken `security:"scheme,type=http,subtype=basic"`
}

type DeleteCustomerProfileEntityAssignmentRequest struct {
	ServerURL  *string
	PathParams DeleteCustomerProfileEntityAssignmentPathParams
	Security   DeleteCustomerProfileEntityAssignmentSecurity
}

type DeleteCustomerProfileEntityAssignmentResponse struct {
	ContentType string
	StatusCode  int64
}

File: SDKs/twilio.com/twilio_trusthub_v1/1.36.0/typescript/src/sdk/models/operations/deletecustomerprofileentityassignment.ts
import { Metadata, SpeakeasyBase } from "../../../internal/utils/utils";
import * as shared from "../shared";
export const DELETECUSTOMERPROFILEENTITYASSIGNMENT_SERVERS = [
	"https://trusthub.twilio.com",
];



export class DeleteCustomerProfileEntityAssignmentPathParams extends SpeakeasyBase {
  @Metadata({ data: "pathParam, style=simple;explode=false;name=CustomerProfileSid" })
  customerProfileSid: string;

  @Metadata({ data: "pathParam, style=simple;explode=false;name=Sid" })
  sid: string;
}


export class DeleteCustomerProfileEntityAssignmentSecurity extends SpeakeasyBase {
  @Metadata({ data: "security, scheme=true;type=http;subtype=basic" })
  accountSidAuthToken: shared.SchemeAccountSidAuthToken;
}


export class DeleteCustomerProfileEntityAssignmentRequest extends SpeakeasyBase {
  @Metadata()
  serverUrl?: string;

  @Metadata()
  pathParams: DeleteCustomerProfileEntityAssignmentPathParams;

  @Metadata()
  security: DeleteCustomerProfileEntityAssignmentSecurity;
}


export class DeleteCustomerProfileEntityAssignmentResponse extends SpeakeasyBase {
  @Metadata()
  contentType: string;

  @Metadata()
  statusCode: number;
}
